code/game/machinery/seed_extractor.dm

```diff
@@ -8,6 +8,43 @@
 
 /obj/structure/machinery/seed_extractor/attackby(obj/item/O as obj, mob/user as mob)
 
+	// Plant bag and other storage containers.
+	if(istype(O,/obj/item/storage))
+		var/obj/item/storage/container = O
+		if(container.contents.len > 0)
+			to_chat(user, SPAN_NOTICE("You start dumping the contents of [container] into [src]."))
+			if(!do_after(user, 1.5 SECONDS, INTERRUPT_ALL, BUSY_ICON_GENERIC))
+				return
+			for(var/obj/item/item in container)
+				if(!istype(item, /obj/item/reagent_container/food/snacks/grown) && !istype(item, /obj/item/grown))
+					to_chat(user, SPAN_WARNING("Cannot get seeds from [item]."))
+					break
+				else
+					if(user.temp_drop_inv_item(item))
+						var/datum/seed/new_seed_type
+						if(istype(item, /obj/item/grown))
+							var/obj/item/grown/F = item
```

Comment:
maints will get mad at one letter var

Reply:
kill the person who wrote the original seed extractor code, but I'll fix it